client/core/core.go

```diff
@@ -6275,8 +6275,13 @@ func (c *Core) prepareTradeRequest(pw []byte, form *TradeForm) (*tradeRequest, e
 	mktID := marketName(form.Base, form.Quote)
 
 	rate, qty := form.Rate, form.Qty
-	if form.IsLimit && rate == 0 {
-		return nil, newError(orderParamsErr, "zero-rate order not allowed")
+	if form.IsLimit {
+		if rate == 0 {
+			return nil, newError(orderParamsErr, "zero-rate order not allowed")
+		}
+		if rate < mktConf.MinimumRate {
+			return nil, newError(orderParamsErr, "order's rate is lower than market's minimum rate. %d < %d", rate, mktConf.MinimumRate)
+		}
```

Comment:
nit: Can the rate be formatted differently. Is it possible to show price? Can be hard to read with all the zeros.

![image](https://github.com/decred/dcrdex/assets/34998433/2bae884b-c405-4e13-b012-b452b3d3f21e)